sql/core/src/test/scala/org/apache/spark/sql/DataFrameComplexTypeSuite.scala

```diff
@@ -26,6 +27,55 @@ import org.apache.spark.sql.test.SharedSQLContext
 class DataFrameComplexTypeSuite extends QueryTest with SharedSQLContext {
   import testImplicits._
 
+  test("create an array") {
+    val df = sparkContext.parallelize(Seq(1, 2), 1).toDF("v")
+    df.selectExpr("Array(v + 3, v + 4)").collect
```

Comment:
To make sure, when the code is eliminated, the simply array creation works well without any exception.
Do you think this is unnecessary?